src/Serilog/Formatting/Display/OutputProperties.cs

```diff
@@ -62,7 +65,7 @@ public static class OutputProperties
         /// </summary>
         /// <param name="logEvent">The log event.</param>
         /// <returns>A dictionary with properties representing the log event.</returns>
-        [Obsolete("Pass the full output template using the other overload.")]
+        [Obsolete("These implementation details of output formatting will not be exposed in a future version.")]
         public static IReadOnlyDictionary<string, LogEventPropertyValue> GetOutputProperties(LogEvent logEvent)
```

Comment:
Just confirming this is the main consumer we are worried about? https://github.com/serilog/serilog-sinks-literate/blob/dev/src/Serilog.Sinks.Literate/Sinks/Literate/LiterateConsoleSink.cs#L88